main.go

```diff
@@ -236,6 +241,7 @@ func ansibleRun() error {
 	runLogger.Infoln("Finding inventory for the current host")
 	inventory, target, err := aCfg.FindInventoryForHost()
 	if err != nil {
+		promAnsibleLastExitCode.Set(404)
```

Comment:
Max value is 255 (-1).
For this you probably want `ENXIO: No such device or address`, which is error code `6`.

Reply:
this is a prom metric so it can be any double float but sure, that makes more sense based on the metric name